plugins/qtcoinrave/CMakeLists.txt

```diff
@@ -28,7 +28,12 @@ if (QT_FOUND AND QT_QTCORE_FOUND AND QT_QTGUI_FOUND AND SOQT_LIBRARY_FOUND AND C
   qt4_wrap_cpp(MOC_OUTPUT_FILES ${QTCOIN_MOCS})
 
   check_include_file(X11/Xlib.h HAVE_X11_XLIB_H)
-
+  if (Boost_MINOR_VERSION LESS 65)

```

Comment:
are you sure you don't have to test Boost MAJOR version here to be 1? For example, in the future Boost 2 might come out, then all this code will be invalid.
thanks